application/auth/forms.py

```diff
@@ -12,5 +13,18 @@ class ForgotPasswordForm(FlaskForm):
 
 
 class LoginForm(FlaskSecurityLoginForm):
-    email = RDUStringField("Email", validators=[Required(message="EMAIL_NOT_PROVIDED")])
-    password = RDUPasswordField("Password", validators=[password_required])
+    email = RDUStringField("Email", validators=[Required(message="Enter your email address")])
+    password = RDUPasswordField("Password", validators=[Required(message="Enter your password")])
+
+    def validate(self):
+        result = super().validate()
+
+        # Check for parent form's processing and override messaging so that we don't leak the existence of a specific
```

Comment:
Remember to move the tech board ticket about this to "Done" once this gets merged.